evc-sumo/src/evc_connector.py

```diff
@@ -70,7 +70,8 @@ def COB_to_traffic_light_status(self, controller_io, evc_phase_id):
         elif controller_io.is_cob_on( (evc_phase_id - 1) + (MAX_INTERSECTION_PHASES * 0) ):
             return 'g'
         else:
-            return None
+            ## if state is not r, y or g then return O which is switched off state
+            return 'O'
```

Comment:
This makes sense, but I think there's a comment below that refers to None being the exception state, which is now '0'. I'd recommend correcting that comment to match the code and then I think we're good!

Reply:
Line 89, I think. Worth a quick check if there's anywhere else too